src/invoices/InvoiceDetails/hooks/useHasPendingOrders.js

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import { ORDER_STATUSES } from '@folio/stripes-acq-components';
+import { useOrders } from '../../../common/hooks';
+
+export const useHasPendingOrders = (orderLinesMap = {}) => {
+  const orderIds = useMemo(() => {
+    return Object.values(orderLinesMap)
+      .map(({ purchaseOrderId }) => purchaseOrderId);
+  }, [orderLinesMap]);
+
+  const {
+    isLoading,
+    orders,
+    isFetched,
+  } = useOrders(orderIds, { skip: !orderIds.length });
```

Comment:
I haven't managed to find the `skip` property in [react-query docs](https://tanstack.com/query/v3/docs/react/reference/useQuery) for `useQuery` options, are you sure you need `skip`?

Reply:
Yeah, you are right. Indeed, `skip` is apollo-query and `enabled` is react-query specific.